velox/functions/sparksql/registration/RegisterMap.cpp

```diff
@@ -22,8 +22,6 @@ extern void registerElementAtFunction(
     bool enableCaching);
 
 void registerSparkMapFunctions(const std::string& prefix) {
-  VELOX_REGISTER_VECTOR_FUNCTION(
```

Comment:
udf_map_allow_duplicates code exists in Presto library, we should move the implement to velox/functions/lib, and add the different logic. If there are too much difference, remove the old udf_map_allow_duplicates, it is only used in Spark

Reply:
It's quite different with Spark's semantics. That's, if there are duplicate keys, it does not deduplicate, but all duplicate keys are retained.